web/src/components/dashboard/SidebarCardGeneric.tsx

```diff
@@ -1,106 +1,99 @@
 import { Content } from "@/components/Content";
-import { SecondaryButton } from "@/components/njwds-extended/SecondaryButton";
+import { PrimaryButton } from "@/components/njwds-extended/PrimaryButton";
 import { UnStyledButton } from "@/components/njwds-extended/UnStyledButton";
 import { Icon } from "@/components/njwds/Icon";
 import { useSidebarCards } from "@/lib/data-hooks/useSidebarCards";
-import { MediaQueries } from "@/lib/PageSizes";
 import { SidebarCardContent } from "@/lib/types/types";
-import useMediaQuery from "@mui/material/useMediaQuery";
-import { ReactElement, ReactNode } from "react";
+import { ReactElement } from "react";
 import { ModifiedContent } from "../ModifiedContent";
 
 type Props = {
   card: SidebarCardContent;
   bodyText: string;
   headerText: string | undefined;
-  preBodyContent?: ReactNode;
+  preBodySpanButtonText?: string;
+  preBodyButtonOnClick?: () => void;
   ctaOnClick?: () => void;
-  layout?: "row" | "column";
 };
 
 export const SidebarCardGeneric = (props: Props): ReactElement => {
   const { hideCard } = useSidebarCards();
-  const isMobile = useMediaQuery(MediaQueries.isMobile);
-
-  const layout = props.layout ?? "column";
 
   const closeSelf = async (): Promise<void> => {
     await hideCard(props.card.id);
   };
 
+  const sideCardClassDictionary: Map<string, string> = new Map();
```

Comment:
pebble, I think it's cleaner to use a dictionary (ordinary object) than a Map/Set to add values.

Reply:
I agree with that. I think this would be more readable and accessible later on the code.

Reply:
done